tests/unit/test_py_vs_cpp.py

```diff
@@ -8,9 +8,14 @@
 from antlr4.Token import CommonToken
 from antlr4.tree.Tree import TerminalNodeImpl
 
+from pytsql import tsql
 from pytsql.grammar import sa_tsql
 from pytsql.tsql import _split
 
+MISSING_PARSER_WARNING_PREFIX = r"Can not find C\+\+ version of the parser"
+
+pytestmark = pytest.mark.filterwarnings(f"ignore:{MISSING_PARSER_WARNING_PREFIX}")
```

Comment:
Is this needed? Doesn't seen to be used in the test.

Reply:
Right, it's not needed anymore, as `test__py_parse__warns` is the only test that warns since my last commit, and that test catches the warning already.